roseau/load_flow/models/transformers/parameters.py

```diff
@@ -97,15 +98,55 @@ def __eq__(self, other: object) -> bool:
             return (
                 self.id == other.id
                 and self.windings == other.windings
-                and np.isclose(self.sn, other.sn)
-                and np.isclose(self.p0, other.p0)
-                and np.isclose(self.i0, other.i0)
-                and np.isclose(self.uhv, other.uhv)
-                and np.isclose(self.ulv, other.ulv)
-                and np.isclose(self.psc, self.psc)
-                and np.isclose(self.vsc, other.vsc)
+                and np.isclose(self._sn, other._sn)
+                and np.isclose(self._p0, other._p0)
+                and np.isclose(self._i0, other._i0)
+                and np.isclose(self._uhv, other._uhv)
+                and np.isclose(self._ulv, other._ulv)
+                and np.isclose(self._psc, other._psc)
+                and np.isclose(self._vsc, other._vsc)
             )
 
+    @property
+    @ureg.wraps("V", (None,), strict=False)
+    def uhv(self) -> Q_:
+        """Phase-to-phase nominal voltages of the high voltages side (V)"""
+        return self._uhv
+
+    @property
+    @ureg.wraps("V", (None,), strict=False)
+    def ulv(self) -> Q_:
+        """Phase-to-phase nominal voltages of the low voltages side (V)"""
+        return self._ulv
+
+    @property
+    def i0(self) -> float:
+        """The nominal voltages of the transformer (VA)"""
+        return self._i0
```

Comment:
You forgot the unit wrapper. And the docstring does not seem correct.

Reply:
Yeah you may consider doing something similar to get a uniform interface for all methods

Reply:
I have added a dimensionless parameter for the `k` (ratio of transformation) of the transformer and for the two unitless electrical arguments of the transformer parameters object. It make sense to transform them into percentage using the new percent unit that I have added. Eg.
```python
k.to("percent") # Q_(0.04, "") ->Q(4, "percent") <=> 4%
```